modules/apps/rate-limiting/keeper/msg_server_test.go

```diff
@@ -0,0 +1,202 @@
+package keeper_test
```

Comment:
Unless I am mistaking, I don't see any tests that verify that the authority is enforced